CMakeLists.txt

```diff
@@ -1,47 +1,20 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(geodetic_utils)
 
-set(CMAKE_BUILD_TYPE Release)
-
-set(CXXFLAGS ${CXXFLAGS} -fPIC)
-# Linux specific flags
-if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
-	set(CXXFLAGS ${CXXFLAGS} -fPIC)
-	add_definitions(-std=c++0x)
-endif()
-
-find_package(catkin REQUIRED COMPONENTS
-  roslib
-  tf
-  geometry_msgs
-  roscpp
-  sensor_msgs
-  cmake_modules
-)
-
-find_package(Eigen REQUIRED)
-
 add_definitions(-std=c++11)
 
-catkin_package(
-  INCLUDE_DIRS include ${Eigen_INCLUDE_DIRS}
-  CATKIN_DEPENDS geometry_msgs roscpp sensor_msgs geometry_msgs
-  DEPENDS Eigen
-)
-
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-  ${Eigen_INCLUDE_DIRS}
-)
+find_package(catkin_simple REQUIRED)
+catkin_simple(ALL_DEPS_REQUIRED)
 
+find_package(Eigen REQUIRED)
+include_directories(${Eigen_INCLUDE_DIRS})
 
-add_executable(set_gps_reference_node src/set_gps_reference_node.cpp)
-add_dependencies(set_gps_reference_node ${catkin_EXPORTED_TARGETS})
+cs_add_executable(set_gps_reference_node src/set_gps_reference_node.cpp)
 target_link_libraries(set_gps_reference_node  ${catkin_LIBRARIES})
```

Comment:
I'm not clear on whether target_link_libraries is necessary for just a binary, with no libraries. Please compare with other nodes (though if it compiles cleanly it should be OK).